association-backend/src/association_membership/domain/repository.py

```diff
@@ -94,10 +95,8 @@ async def create_stripe_portal_session_url(
     async def save_subscription(self, subscription: Subscription) -> Subscription:
         await subscription.update()
 
-        for (
-            stripe_subscription_payment
-        ) in subscription._add_stripe_subscription_payment:
-            await stripe_subscription_payment.payment.save()
-            await stripe_subscription_payment.save()
+        for add_payment in subscription._add_payments:
```

Comment:
shall we rename `_add_payments` to `payments_to_add` or `new_payments`?

Reply:
I can rename it `_payments_to_add` (see my answer above on why I prefer the `_`)